common/credentials/src/coconut/bandwidth/freepass.rs

```diff
@@ -1,105 +1,27 @@
 // Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::coconut::utils::scalar_serde_helper;
 use crate::error::Error;
 use nym_api_requests::coconut::FreePassRequest;
-use nym_credentials_interface::{
-    hash_to_scalar, Attribute, BlindedSignature, CredentialSigningData, PublicAttribute,
-};
+use nym_credentials_interface::{BlindedSignature, CredentialSigningData};
 use nym_validator_client::signing::AccountData;
 use serde::{Deserialize, Serialize};
-use time::{Duration, OffsetDateTime, Time};
 use zeroize::{Zeroize, ZeroizeOnDrop};
 
-pub const MAX_FREE_PASS_VALIDITY: Duration = Duration::WEEK; // 1 week
-
 #[derive(Debug, Zeroize, ZeroizeOnDrop, Serialize, Deserialize)]
-pub struct FreePassIssuedData {
-    /// the plain validity value of this credential expressed as unix timestamp
-    #[zeroize(skip)]
-    expiry_date: OffsetDateTime,
-}
+pub struct FreePassIssuedData {}
```

Comment:
doesn't freepass have an explicit expiration date?

Reply:
It's not needed in ecash, as any ticket book has an expiration date. The expiration date that was there was moved up in the struct that is using `FreePassIssuedData`.
Freepasses have nothing that normal ticket books do not have